lib/ProductOpener/Display.pm

```diff
@@ -5736,12 +5740,21 @@ sub search_and_export_products ($request_ref, $query_ref, $sort_by) {
 	return;
 }
 
+sub escape_char($s, $char) {
```

Comment:
Looks good. Maybe this function (and escape_single_quote) could be moved to Text.pm